pkg/dependency/dotnet_aspnetcore_test.go

```diff
@@ -257,6 +257,7 @@ func testDotnetASPNETCore(t *testing.T, when spec.G, it spec.S) {
 				SHA:             "some-sha256",
 				ReleaseDate:     "2020-02-20T00:00:00Z",
 				DeprecationDate: "2050-02-20T00:00:00Z",
+				CPE:             "cpe:2.3:a:microsoft:.net_core:2.0.1:*:*:*:*:*:*:*",
```

Comment:
It looks like ASPNet has a [separate CPE notation](https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=asp.net_core), though it doesn't seem to include the patch version. Did you see this?

Reply:
Nope, we missed that one

Reply:
It looks like ASPNet always uses `_core`, though, unlike regular DotNet, based on searching for [`cpe:2.3:a:microsoft:asp.net_core:*:*`](https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=cpe%3A2.3%3Aa%3Amicrosoft%3Aasp.net_core%3A*%3A*)

Reply:
Fixed in latest commit